tsl/test/expected/backup.out

```diff
@@ -0,0 +1,124 @@
+-- LICENSE-TIMESCALE for a copy of the license.
+-- Need to be super user to create extension and add data nodes
+\c :TEST_DBNAME :ROLE_CLUSTER_SUPERUSER;
+\unset ECHO
+psql:include/remote_exec.sql:5: NOTICE:  schema "test" already exists, skipping
+\set MY_DB1 :TEST_DBNAME _1
+\set MY_DB2 :TEST_DBNAME _2
+\set MY_DB3 :TEST_DBNAME _3
+SELECT * FROM add_data_node('data_node_1', host => 'localhost', database => :'MY_DB1');
+  node_name  |   host    | port  |  database   | node_created | database_created | extension_created 
+-------------+-----------+-------+-------------+--------------+------------------+-------------------
+ data_node_1 | localhost | 55432 | db_backup_1 | t            | t                | t
+(1 row)
+
+SELECT * FROM add_data_node('data_node_2', host => 'localhost', database => :'MY_DB2');
+  node_name  |   host    | port  |  database   | node_created | database_created | extension_created 
+-------------+-----------+-------+-------------+--------------+------------------+-------------------
+ data_node_2 | localhost | 55432 | db_backup_2 | t            | t                | t
+(1 row)
+
+SELECT * FROM add_data_node('data_node_3', host => 'localhost', database => :'MY_DB3');
+  node_name  |   host    | port  |  database   | node_created | database_created | extension_created 
+-------------+-----------+-------+-------------+--------------+------------------+-------------------
+ data_node_3 | localhost | 55432 | db_backup_3 | t            | t                | t
+(1 row)
+
+GRANT USAGE ON FOREIGN SERVER data_node_1, data_node_2, data_node_3 TO PUBLIC;
+-- Import testsupport.sql file to data nodes
+\unset ECHO
+-- test NULL (STRICT function)
+SELECT create_distributed_restore_point(NULL);
+ create_distributed_restore_point 
+----------------------------------
+(0 rows)
+
+-- test long restore point name (>= 64 chars)
+\set ON_ERROR_STOP 0
+SELECT create_distributed_restore_point('0123456789012345678901234567890123456789012345678901234567890123456789');
+ERROR:  restore point name is too long
+\set ON_ERROR_STOP 1
+-- test super user check
+\set ON_ERROR_STOP 0
+SET ROLE :ROLE_1;
+SELECT create_distributed_restore_point('test');
+ERROR:  must be superuser to create restore point
+RESET ROLE;
+\set ON_ERROR_STOP 1
+-- make sure 2pc are enabled
+SET timescaledb.enable_2pc = false;
+\set ON_ERROR_STOP 0
+SELECT create_distributed_restore_point('test');
+ERROR:  two-phase commit transactions are not enabled
+\set ON_ERROR_STOP 1
+-- make sure wal_level is replica or logical
+SHOW wal_level;
+ wal_level 
+-----------
+ replica
+(1 row)
+
+-- make sure not in recovery mode
+SELECT pg_is_in_recovery();
+ pg_is_in_recovery 
+-------------------
+ f
+(1 row)
+
+-- test on single node
+CREATE DATABASE dist_rp_test;
+\c dist_rp_test :ROLE_CLUSTER_SUPERUSER
+SET client_min_messages TO ERROR;
+CREATE EXTENSION timescaledb;
+\set ON_ERROR_STOP 0
+SELECT create_distributed_restore_point('test');
```

Comment:
What happens if this is called via `distributed_exec` from the access node?

Reply:
It will fail as expected, added the case